src/test/scala/thurloe/service/ThurloeServiceSpec.scala

```diff
@@ -365,9 +367,9 @@ class ThurloeServiceSpec extends AnyFunSpec with ScalatestRouteTest {
 
     it("should handle sam users with differing ids") {
       // prepare values and mocks
-      val userSamId = "samId"
-      val userSubjectId = "subjectId"
-      val userB2cId = "b2cId"
+      val userSamId = UUID.randomUUID().toString
+      val userSubjectId = UUID.randomUUID().toString
+      val userB2cId = UUID.randomUUID().toString
```

Comment:
I am suspicious that database state isnt reset between tests so i made this change